worlds/stardew_valley/__init__.py

```diff
@@ -64,6 +64,12 @@ class StardewValleyWorld(World):
     item_name_to_id = {name: data.code for name, data in item_table.items()}
     location_name_to_id = {name: data.code for name, data in location_table.items()}
 
+    item_name_groups = {group.name.replace("_", " ").title() + (" Group" if group.name.replace("_", " ").title()
+                                                                in item_table else ""):
+                        [item.name for item in items] for group, items in items_by_group.items()}
+    location_name_groups = {group.name.replace("_", " ").title(): [item.name for item in locations]
```

Comment:
this should be `[location.name for location in locations]`

Reply:
Right right I saw that and intended to fix it and didn't, oops

Reply:
```suggestion
    location_name_groups = {group.name.replace("_", " ").title(): [location.name for location in locations]
```